backend/routes/searchWithFilterDpsUserRouter.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express')
```

Comment:
This was copy and pasted more or less so didn't do a unit test, will add one in the next PR